llama_stack/providers/tests/ci_test_config.yaml

```diff
@@ -0,0 +1,59 @@
+inference:
+  tests:
+  - inference/test_vision_inference.py::test_vision_chat_completion_streaming
+  - inference/test_vision_inference.py::test_vision_chat_completion_non_streaming
+  - inference/test_text_inference.py::test_structured_output
+  - inference/test_text_inference.py::test_chat_completion_streaming
+  - inference/test_text_inference.py::test_chat_completion_non_streaming
+  - inference/test_text_inference.py::test_chat_completion_with_tool_calling
+  - inference/test_text_inference.py::test_chat_completion_with_tool_calling_streaming
+
+  fixtures:
+   provider_fixtures:
```

Comment:
this is a confusing thing -- why are there `provider_fixtures` within `fixtures`? also it is unclear why we have a "default_fixture_param_id" and what it indicates. Can you explain a bit?